constants/network.js

```diff
@@ -1,51 +1,44 @@
-const metadataTypes = {
-  // metadata file will contain all individual metadata files (common for eth$, sol$)
-  // no rarities at all
-  basic: 0,
-  // metadata file will contain only rarity data for traits & attributes (common for egld$)
-  // if rarityAlgorithm provided, individual metadata file will also contain rarity data
-  rarities: 1,
-};
+const { METADATA } = require("../constants/metadata");
+const { RARITY } = require("../constants/rarity");
 
-const rarityAlgorithms = {
-  none: 0,
-  JaccardDistances: 1, // most accurate / recommended
-  TraitRarity: 2,
-  StatisticalRarity: 3,
-  TraitAndStatisticalRarity: 4, // TraitRarity & StatisticalRarity combined
+const defaults = {
+  startIdx: 1,
+  jsonDirPrefix: "",
+  mediaDirPrefix: "",
+  mediaFilePrefix: "",
+  metadataFileName: "_metadata.json",
+  metadataType: METADATA.basic,
+  rarityAlgorithm: RARITY.none,
+  includeRank: false,
 };
 
 const NETWORK = {
-  egld: {
-    name: "egld",
-    startIdx: 1,
-    metadataFileName: "_metadata.json",
-    metadataType: metadataTypes.rarities,
-    rarityAlgorithm: rarityAlgorithms.JaccardDistances,
-    includeRank: true
-  },
   eth: {
+    ...defaults,
     name: "eth",
-    startIdx: 1,
     jsonDirPrefix: "json/",
     mediaDirPrefix: "media/",
     mediaFilePrefix: "$",
-    metadataFileName: "_metadata.json",
-    metadataType: metadataTypes.basic,
   },
   sol: {
+    ...defaults,
     name: "sol",
     startIdx: 0,
     jsonDirPrefix: "json/",
     mediaDirPrefix: "media/",
     mediaFilePrefix: "$",
-    metadataFileName: "_metadata.json",
-    metadataType: metadataTypes.basic,
+  },
+  egld: {
+    ...defaults,
+    name: "egld",
+    metadataType: METADATA.rarities,
+    rarityAlgorithm: RARITY.JaccardDistances,
+    includeRank: true,
   },
 };
 
 module.exports = {
   NETWORK,
-  metadataTypes,
-  rarityAlgorithms,
+  METADATA,
```

Comment:
We should not import/export extra items in `network.js`, import them from `metadata.js` associated files where needed